src/storage/db_storage.cpp

```diff
@@ -251,6 +251,30 @@ std::vector<std::shared_ptr<DagBlock>> DbStorage::getDagBlocksAtLevel(level_t le
   return res;
 }
 
+void DbStorage::updateDagBlockCounters(Batch& write_batch, vector<DagBlock> blks) {
+  // Lock is needed since we are editing some fields
+  lock_guard<mutex> u_lock(dag_blocks_mutex_);
+  for (auto const& blk : blks) {
+    auto level = blk.getLevel();
+    std::string blocks = getBlocksByLevel(level);
+    if (blocks == "") {
+      blocks = blk.getHash().toString();
+    } else {
+      blocks = blocks + "," + blk.getHash().toString();
```

Comment:
why we are storing it as string? isn't it better to have rlp array of block hashes or something?

Reply:
Yes, this should be modified. I would do it later as a separate PR and change all the places we use this.